### PR TITLE
Feature/add eslint config

### PR DESCRIPTION
**Pull request by conandk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,10 +40,10 @@
     "build": "tsc",
     "build:web": "webpack --config ./webpack/webpack.prod.js",
     "start": "node ./server",
-    "prettier": "prettier --write ./**/*.{js,ts,tsx,md}",
+    "prettier": "./node_modules/.bin/prettier --write ./**/*.{js,ts,tsx,md}",
     "test": "./node_modules/.bin/jest",
-    "lint": "eslint . --ext .ts,.tsx",
-    "lint:fix": "eslint . --ext .ts,.tsx --fix",
+    "lint": "./node_modules/.bin/eslint . --ext .ts,.tsx",
+    "lint:fix": "./node_modules/.bin/eslint . --ext .ts,.tsx --fix",
```
Comment (davidnguyen179) [inline package.json:46]: Do we really need this, since we already had pretter?

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+coverage
+
+.vscode
+
+.githooks
+
+.github
+
+dist
```
Comment (davidnguyen179) [inline .eslintignore:11]: how about `build` dir?

File: pages/app.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import ImageUploading from '../src';
 
-export function App() {
+export default function App() {
```
Comment (davidnguyen179) [inline pages/app.tsx:4]: For the sake of intelligent feature. I recommend use `export` not `export default`

**Comment by davidnguyen179** [comment]
it failed unit test @conandk

File: package.json (hunk 1 above)
Comment (conandk) [reply package.json:46]: we need it. ESLint to fix coding errors and Prettier to fix stylistic errors

File: .eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  parserOptions: {
+    project: './tsconfig.eslint.json',
+  },
+  rules: {
+    'react/jsx-props-no-spreading': 'off',
+    'react/prop-types': 'off',
+    'no-unused-expressions': 'off',
+    '@typescript-eslint/no-unused-expressions': 'off',
+    'no-continue': 'off',
+    'no-await-in-loop': 'off',
```
Comment (conandk) [inline .eslintrc.js:21]: @vutoan266 , temp off this rule. Can you help to fix it?

**Pull request merged**